src/pages/Profile.js

```diff
@@ -465,48 +573,82 @@ const Profile = ({ queryParams }) => {
                 }, ['user guide.'])
               ])
             ]),
-            div({ style: { margin: '1rem' } }, [proxyGroup]),
-
-            sectionTitle('Location'),
+            div({ style: { margin: '1rem' } }, proxyGroup ? proxyGroup : 'Loading...')
+          ])
+        ]),
 
-            line([
-              textField('programLocationCity', 'City'),
-              textField('programLocationState', 'State')
-            ]),
-            line([
-              textField('programLocationCountry', 'Country')
-            ]),
+        sectionTitle('What is your area of research? (multi-select)'),
```

Comment:
Instead of "multi-select", maybe add a paragraph below this that says something like "Check all that apply."?

Reply:
Updated in 5103bd555bfcbae02e9ae3e56748bfeb13d26155